README.rst

```diff
@@ -39,165 +39,173 @@ Python 3.8+
-        "spline": imputer_spline,
-        "shuffle": imputer_shuffle,
-        "residuals": imputer_residuals,
-        "OU": imputer_ou,
-        "TSOU": imputer_tsou,
-        "TSMLE": imputer_tsmle,
-        "RPCA": imputer_rpca,
-        "RPCA_opti": imputer_rpca_opti,
-        "locf": imputer_locf,
-        "nocb": imputer_nocb,
-        "knn": imputer_knn,
-        "ols": imputer_regressor,
-        "mice_ols": imputer_mice,
-    }
-
-It is possible to define a parameter dictionary for an imputer with three pieces of information: min, max and type. The aim of the dictionary is to determine the optimal parameters for data imputation. Here, we call this dictionary ``dict_config_opti``.
-
-.. code-block:: python
-
-    search_params = {
-        "RPCA_opti": {
-            "tau": {"min": .5, "max": 5, "type":"Real"},
-            "lam": {"min": .1, "max": 1, "type":"Real"},
-        }
-    }
-
-Then with the comparator function in ``from qolmat.benchmark import comparator``, we can compare the different imputation methods. This **does not use knowledge on missing values**, but it relies data masking instead. For more details on how imputors and comparator work, please see the following `link <https://qolmat.readthedocs.io/en/latest/explanation.html>`_.
-
-.. code-block:: python
-
-    from qolmat.benchmark import comparator
-
-    generator_holes = missing_patterns.EmpiricalHoleGenerator(n_splits=4, ratio_masked=0.1)
-
-    comparison = comparator.Comparator(
-        dict_imputers,
-        ['y'],
-        generator_holes = generator_holes,
-        metrics = ["mae", "wmape", "KL_columnwise", "ks_test", "energy"],
-        n_calls_opt = 10,
-        dict_config_opti = dict_config_opti,
-    )
-    results = comparison.compare(df_with_nan)
+  import matplotlib.pyplot as plt
+  # visualise
+  dfs_imputed = {name: imp.fit_transform(df_with_nan) for name, imp in dict_imputers.items()}
+  plt.figure(figsize=(13,3))
+  for (name, df_imputed), color in zip(dfs_imputed.items(), ["tab:green", "tab:blue", "tab:red"]):
+      plt.plot(df_imputed, ".", c=color, label=name)
+  plt.plot(df_with_nan, ".", c="k", label="original")
+  plt.legend()
+  plt.grid()
+  plt.ylabel("values")
+  plt.show()
+
+.. image:: https://raw.githubusercontent.com/Quantmetry/qolmat/main/docs/images/readme_imputation_plot.png
+    :align: center
 
-We can observe the benchmark results.
 
-.. image:: https://raw.githubusercontent.com/Quantmetry/qolmat/main/docs/images/readme2.png
-    :align: center
+📘 Documentation
+================
 
-Finally, we keep the best ``TSMLE`` imputor we represent.
+The full documentation can be found `on this link <https://qolmat.readthedocs.io/en/latest/>`_.
 
-.. code-block:: python
+**How does Qolmat work ?**
 
-    dfs_imputed =  imputer_tsmle.fit_transform(df_with_nan)
+Qolmat simplifies the selection process of a data imputation algorithm. It does so by comparing of various methods based on different evaluation metrics.
```

Comment:
L'anglais de ce paragraphe comporte quelques erreurs mineures de grammaire et de formulation. Voici une version corrigée :

"Qolmat simplifies the process of selecting a data imputation algorithm by comparing various methods based on different evaluation metrics. It is compatible with scikit-learn. The evaluation and comparison are based on the standard approach of selecting certain observations, setting their status to missing, and comparing their imputed values with their true values."

Les modifications apportées incluent l'ajout de l'article "the" avant "evaluation" et "comparison" pour les rendre spécifiques, ainsi que la correction de la préposition "of" à "by" pour indiquer la méthode utilisée pour la comparaison. De plus, "set their status to missing" a été modifié pour une meilleure compréhension de la phrase.